app/models/client.rb

```diff
@@ -2,23 +2,24 @@
 #
 # Table name: clients
 #
-#  id                           :bigint           not null, primary key
-#  attention_needed_since       :datetime
-#  current_sign_in_at           :datetime
-#  current_sign_in_ip           :inet
-#  failed_attempts              :integer          default(0), not null
-#  last_incoming_interaction_at :datetime
-#  last_interaction_at          :datetime
-#  last_sign_in_at              :datetime
-#  last_sign_in_ip              :inet
-#  locked_at                    :datetime
-#  login_requested_at           :datetime
-#  login_token                  :string
-#  routing_method               :integer
-#  sign_in_count                :integer          default(0), not null
-#  created_at                   :datetime         not null
-#  updated_at                   :datetime         not null
-#  vita_partner_id              :bigint
+#  id                                          :bigint           not null, primary key
+#  attention_needed_since                      :datetime
+#  current_sign_in_at                          :datetime
+#  current_sign_in_ip                          :inet
+#  failed_attempts                             :integer          default(0), not null
+#  first_unanswered_incoming_correspondence_at :datetime
```

Comment:
i used _correspondence_ here because originally product had asked only to track based on incoming messages (not documents) but since the intention is to compare as close to apples-to-apples with needs attention (other than using the manual resolve button) im wondering if "first_unanswered_incoming_interaction_at" makes more sense?

Reply:
True. What about something like "sla_response_needed_since"?